packages/office-ui-fabric-react/src/components/Button/ButtonCore/ButtonCore.scss

```diff
@@ -58,63 +52,3 @@
   vertical-align: top;
   display: inline-block;
 }
-
-// TODO: Remove Hero Styles once fully deprecated
-
-//== Modifier: Hero button
-//
-.ms-Button--hero {
-  background-color: transparent;
-  border: 0;
-  height: auto;
-
-  .ms-Button-icon {
-    color: $ms-color-themePrimary;
-    display: inline-block;
-    @include margin-right(8px);
-    padding-top: 5px;
-    font-size: $ms-icon-size-l;
-    line-height: 1; // FireFox vertical offset
-  }
-
-  .ms-Button-label {
-    color: $ms-color-themePrimary;
-    font-size: $ms-font-size-xl;
-    font-weight: $ms-font-weight-light;
-    vertical-align: top;
-  }
-
-  &:hover,
-  &:focus {
-    background-color: transparent;
-
-    .ms-Button-icon {
-      color: $ms-color-themeDark;
-    }
-
-    .ms-Button-label {
-      color: $ms-color-themeDarker;
-    }
-  }
-
-  &:active {
-    .ms-Button-icon {
-      color: $ms-color-themePrimary;
-    }
-
-    .ms-Button-label {
-      color: $ms-color-themePrimary;
-    }
-  }
-
-  &:disabled,
-  &.is-disabled {
-    .ms-Button-icon {
-      color: $ms-color-neutralTertiaryAlt;
```

Comment:
Why did you remove the disabled stuff from here? It really seems to cut down on code duplication. Now in every other *Button.scss you have to have &.is-disabled with `color: $ms-color-neutralTertiaryAlt;` Additionally you'll have to change it in all those places if it every changes and it makes it harder for me to subclass easily, I'll have to rewrite the css every time.